langchain/chat_models/base.py

```diff
@@ -93,6 +94,8 @@ def generate(
         generations = [res.generations for res in results]
         output = LLMResult(generations=generations, llm_output=llm_output)
         run_manager.on_llm_end(output)
+        if run_manager:
+            output.run = RunInfo(run_id=run_manager.run_id)
```

Comment:
Should this be RUN_KEY

Reply:
unfortunately you cannot set underscore keys in pydantic objects, i.e. `LLMResult`

Reply:
Wow these python libraries are something else

Reply:
Could set the alias then serialize with `by_alias=True` though maybe that's even more complicated

Reply:
yeah... seems more complex than it needs to imo